src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockBuilder.cs

```diff
@@ -134,6 +134,6 @@ public override void Reset()
         /// <summary>
         /// The starting <see cref="SourceLocation"/> of the tag helper.
         /// </summary>
-        public SourceLocation Start { get; private set; }
+        public SourceLocation Start { get; set; }
```

Comment:
Is this change needed only for testing? If so shouldn't this just be `internal`?